svc-lighthouse-api/src/test/resources/empty-bundle.json

```diff
@@ -0,0 +1,20 @@
+{
+  "resourceType": "Bundle",
+  "type": "searchset",
+  "total": 0,
+  "link": [
+    {
+      "relation": "first",
+      "url": "https://sandbox-api.va.gov/resource"
+    },
+    {
+      "relation": "self",
+      "url": "https://sandbox-api.va.gov/resource"
+    },
+    {
+      "relation": "last",
+      "url": "https://sandbox-api.va.gov/resource"
+    }
+  ],
+  "entry": []
```

Comment:
Is this a real return from Lighthouse FHIR API?

Reply:
Not exactly but a similar empty bundle is observed in production causing 500 errors which this PR fixes.